src/Request/AddTrackToPlaylistRequest.php

```
@@ -16,11 +17,18 @@ public function __construct(private TrackToPlaylistDto $trackToPlaylistDto, priv
 
     public function execute(): BaseSpotifyResponse
     {
-        return SpotifyHttpClient::postApiCall(
+        $response = SpotifyHttpClient::postApiCall(
            'v1/playlists/'.$this->trackToPlaylistDto->playlistId.'/tracks',
             $this->headers->toArray(),
             $this->trackToPlaylistDto->toArray()
         );
+        if ($response instanceof BaseSpotifyResponse) {
+            return $response;
+        } elseif ($response instanceof ResponseInterface) {
+            return new BaseSpotifyResponse($response->getData());
+        } else {
+            return new BaseSpotifyResponse([]);
```

Should throw an exception here rather than an empty response